.gitignore

```diff
@@ -7,6 +7,8 @@
 
 # testing
 /coverage
+.idea
```

Comment:
could you please add  `.end` and `.env.* ` as well
i noticed .env has been pushed, that was my mistake